src/Helper/YahooHelper.hs

```diff
@@ -119,37 +123,51 @@ instance FromField YUTCTime where
     x <- parseTimestamp "%Y-%m-%d" (C.unpack (C.fromStrict u))
     pure (YUTCTime x)
 
-getYahooData :: Text -> ExceptT YahooException IO C.ByteString
-getYahooData ticker =
+
+getYahooData :: Text -> ExceptT YL.YahooException IO C.ByteString
+getYahooData ticker = do
+  -- endDate <- getCurrentTime
```

Comment:
‘endDate <- liftIO getCurrentTime’
Nice try with the date in the future :)

Reply:
Updated the helper - liftIO is commited and now you could merge!